contracts/modules/AaveModule.sol

```diff
@@ -34,7 +34,11 @@ contract AaveModule is BaseModule {
     ///@notice deposit a position in an Aave lending pool
     ///@param positionManager address of the position manager
     ///@param tokenId id of the Uniswap position to deposit
-    function depositIfNeeded(address positionManager, uint256 tokenId) public activeModule(positionManager, tokenId) {
+    function depositIfNeeded(address positionManager, uint256 tokenId)
```

Comment:
We were requested to add the check also in other parts of the project:

-  modules/AaveModule.sol - depositIfNeeded() in line 37
-  modules/AaveModule.sol - withdrawIfNeeded() in line 55
-  modules/AutoCompoundModule.sol - autoCompoundFees() in line 30
-  modules/IdleLiquidityModule.sol - rebalance() in line 31